tests/command_count_test.go

```diff
@@ -0,0 +1,34 @@
+package tests
+
+import (
+	"fmt"
+	"testing"
+	"gotest.tools/v3/assert"
+)
+
+func TestCommandCount(t *testing.T) {
+	connection := getLocalConnection()
+	responseValue := fireCommand(connection, "COMMAND COUNT")
+	responseCount  := responseValue.(int64)
+	if responseValue == nil {
+		t.Fail()
+	}
+
+	assert.Assert(t, responseCount > 0,
+		fmt.Sprintf("Unexpected number of CLI commands found. expected greater than 0, %d found", responseCount))
+}
+
+func getCommandCount(t *testing.B) {
+	connection := getLocalConnection()
```

Comment:
Is it a good idea to include connection establishment as a part of the benchmark code? Maybe the connection should be created once in the caller and be reused across function calls for the benchmark.

Reply:
I have updated to reuse connection.